template/report.html.tmpl

```diff
@@ -9,14 +9,19 @@
   <meta name="description" content="Sentry SDK Benchmark Report" />
   <meta name="author" content="Sentry (sentry.io)" />
 
+  {{ range .ReportCSS }}
   <style type="text/css">
```

Comment:
The `type` attribute here and in `<script>` is obsolete, right?

Reply:
Yup, will remove